_posts/2016-09-00-edsls-part-1.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: It's programs all the way down
+
+meta:
+  nav: blog
+  author: adelbertc
+  pygments: true
+---
+
+*This is the first of a series of articles on "EDSLs in Scala."*
```

Comment:
My current plan only is to mostly explore MTL/tagless-final and `Free`(`Applicative`).. I'm not sure what the correct terminology for this would be. Any suggestions? :D